docket/plugins/config_scripts.py

```diff
@@ -0,0 +1,167 @@
+from __future__ import annotations
+
+import asyncio
+from difflib import get_close_matches
+from typing import cast
+
+import asyncpg
+import crescent
+import hikari
+
+from docket.config import CONFIG
+from docket.database.models.guild import Guild
+from docket.database.models.script import Script
+from docket.errors import DocketBaseError, ScriptNotFound
+from docket.plugins._checks import has_guild_perms
+from docket.plugins._group import docket_group
+
+plugin = crescent.Plugin(
+    "config-scripts", command_hooks=[has_guild_perms(hikari.Permissions.MANAGE_GUILD)]
+)
+scripts = docket_group.sub_group("scripts", "Manage scripts")
+
+
+async def wait_for_script(ctx: crescent.Context) -> str | None:
+    def predicate(message: hikari.GuildMessageCreateEvent) -> bool:
```

Comment:
i think this can be more descriptive

Reply:
@Lunarmagpie `predicate` is the term used by hikari's `wait_for` method.

Reply:
ig its fine then